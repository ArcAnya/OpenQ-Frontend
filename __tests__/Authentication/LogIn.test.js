import React from 'react';
import LogIn from '../../components/Authentication/LogIn';
import renderer from 'react-test-renderer';

describe('Login Snapshot', () => {
  it('should match DOM Snapshot', () => {
    const tree = renderer.create(<LogIn />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
