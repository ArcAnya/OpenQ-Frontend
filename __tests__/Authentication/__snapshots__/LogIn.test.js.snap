// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Snapshot should match DOM Snapshot 1`] = `
<button
  className="flex justify-center btn-default whitespace-nowrap w-full"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onTouchStart={[Function]}
>
  <div
    className="flex flex-row items-center justify-center space-x-3"
  >
    <div>
      Log in
    </div>
  </div>
</button>
`;
